examples/evaluators/evaluate_on_regressor.py

```diff
@@ -0,0 +1,28 @@
+from mlflow.models.evaluation import evaluate, EvaluationDataset
+import mlflow
+from sklearn.datasets import load_boston
+from sklearn.linear_model import LinearRegression
+
+mlflow.sklearn.autolog()
+
+X, y = load_boston(return_X_y=True)
+dataset = EvaluationDataset(data=X, labels=y, name='boston', feature_names=[
+    f'feature_{i}' for i in range(13)
```

Comment:
for feature_names, do we want to show the actual names from the source X df for the example?
```python
boston_data = load_boston()
dataset = EvaluationDataset(data=boston_data.data, labels=boston_data.target, name='boston', feature_names=boston_data.feature_names)
```